packages/azure/src/loadBalancer/details/loadBalancerDetail.html

```diff
@@ -27,7 +27,7 @@ <h3 class="horizontal middle space-between flex-1" select-on-dbl-click>{{loadBal
             Load Balancer Actions <span class="caret"></span>
           </button>
           <ul class="dropdown-menu" uib-dropdown-menu role="menu">
-            <li><a href ng-click="ctrl.editLoadBalancer()">Edit Load Balancer</a></li>
+            <li class="disabled"><a>Edit Load Balancer</a></li>
```

Comment:
I wonder if the methods should just return false instead so the context isn't lost or if we ever want to enable it for some reason?